Error running tests (Error: Unexpected path .DS_Store)

I'm trying to run tests on a Mac and I'm getting the following:

```
 ▲ ~/src/DefinitelyTyped npm test                                                                               22s  update-mapkit :: 20m :: ⬢

> definitely-typed@0.0.3 test /Users/cwd/src/DefinitelyTyped
> node node_modules/types-publisher/bin/tester/test.js --run-from-definitely-typed

Clean data
Clean logs
Clean output
Using local Definitely Typed at /Users/cwd/src/DefinitelyTyped.
Parsing definitions...
Error: Unexpected path .DS_Store
    at validatePath (/Users/cwd/src/DefinitelyTyped/node_modules/types-publisher/src/get-definitely-typed.ts:256:15)
    at DiskFS.getPath (/Users/cwd/src/DefinitelyTyped/node_modules/types-publisher/src/get-definitely-typed.ts:217:13)
    at DiskFS.isDirectory (/Users/cwd/src/DefinitelyTyped/node_modules/types-publisher/src/get-definitely-typed.ts:227:33)
    at /Users/cwd/src/DefinitelyTyped/node_modules/types-publisher/src/parse-definitions.ts:35:131
    at /Users/cwd/src/DefinitelyTyped/node_modules/types-publisher/src/util/util.ts:56:34
    at initArray (/Users/cwd/src/DefinitelyTyped/node_modules/types-publisher/src/util/util.ts:114:18)
    at nAtATime (/Users/cwd/src/DefinitelyTyped/node_modules/types-publisher/src/util/util.ts:51:23)
    at Object.filterNAtATimeOrdered (/Users/cwd/src/DefinitelyTyped/node_modules/types-publisher/src/util/util.ts:88:42)
    at Object.parseDefinitions [as default] (/Users/cwd/src/DefinitelyTyped/node_modules/types-publisher/src/parse-definitions.ts:35:32)
    at main (/Users/cwd/src/DefinitelyTyped/node_modules/types-publisher/src/tester/test.ts:24:5)
npm ERR! Test failed.  See above for more details.
```

--- a/src/get-definitely-typed.ts
+++ b/src/get-definitely-typed.ts
@@ -225,7 +225,7 @@ class DiskFS implements FS {
     }
 
     readdir(dirPath?: string): ReadonlyArray<string> {
-        return assertSorted(readdirSync(this.getPath(dirPath))).filter(name => name !== ".DS_STORE");
+        return assertSorted(readdirSync(this.getPath(dirPath))).filter(name => name !== ".DS_Store");
     }
 
     isDirectory(dirPath: string): boolean {

